tiny-menu.el

```diff
@@ -104,9 +124,7 @@ of the key binding declarations for your menus.  A key binding
 declaration now looks like:
 
 `(define-key some-map \"<key>\" (tiny-menu-item \"my-menu\"))'."
-  (lambda ()
-    (interactive)
-    (tiny-menu item)))
+  `(lambda () (interactive) (tiny-menu ,item)))
```

Comment:
Please return this to the regular function implementation as Steve Purcell recommended.